src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -253,23 +253,19 @@ def get_workspace_triad():
 
 
 def list_deployment_connections(
-    subscription_id,
-    resource_group_name,
-    workspace_name,
+    subscription_id="",
```

Comment:
The doc examples set the sub, rg, ws to None. Is it better to use none here?

Reply:
https://github.com/microsoft/promptflow/pull/2310/files#diff-03b8cd09e8a52b3a83b7e7a4578566a01b724b35e6c8bbc934bad277d3c21b7a

Reply:
updated